protocol/hello/hello_test.go

```diff
@@ -230,4 +268,7 @@ func TestHelloMultiBlock(t *testing.T) {
 
 	msc1.AssertExpectations(t)
 	msc2.AssertExpectations(t)
+
+	assert.True(t, trackerA.contains(b.ID()))
+	assert.True(t, trackerB.contains(a.ID()))
```

Comment:
If refactored as I suggest, I expect this diff to disappear. It's an issue for integration testing (probably sufficiently covered by daemon tests for now), not for hello unit testing.